src/Knub.ts

```diff
@@ -365,7 +370,7 @@ export class Knub<
       }
 
       // Only load the guild if we're actually in the guild
-      if (!this.client.guilds.cache.has(guildId)) {
+      if (!this.client.guilds.resolve(guildId as any)) {
```

Comment:
Change the `guildId` type in the function argument to `Snowflake` instead

Reply:
Thank you!